spec/factories.rb

```diff
@@ -87,7 +87,7 @@
 
   factory :transaction do
     bank_account
-    category
+    category_id { create(:category, organization: bank_account.organization).id }
```

Comment:
@Gataniel don't use it such way. Use association. If you have some problems with it then ask me for help.